canonical-websites/www.ubuntu.com
# Cookie bar

## Done

Updated the cookie bar to the latest design and sandboxed the changes to the v1 rollout

## QA

If you do not see the cookie bar on `/about` then add `state('open');` to line 114 of `scratch-v1.js`.

Check the cookie bar against [the design](https://cloud.githubusercontent.com/assets/9105999/23611548/cf5fd186-026f-11e7-89e4-693e4f1b6d08.png) and ensure that it works correctly at all viewports.
